### PR TITLE
Test directory w watchify and more for actively developing

### PR DESCRIPTION
**Pull request by joelgriffith**
Following feedback from https://github.com/graphql/graphiql/pull/197, this adds a `test` directory and a small server to load up a graphiql interface to actively test changes. Currently proxies graphql requests over to swapi-graphql (however the introspection query fails...).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,13 +32,14 @@
     "mocha": "--full-trace --require resources/mocha-bootload src/**/*-test.js"
   },
   "scripts": {
```
Comment (joelgriffith) [inline package.json]: Alphabetizing

File: test/server.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/**
```
Comment (joelgriffith) [inline test/server.js]: Bundling happens here to help consolidate files

File: test/index.html [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+          window.location.origin =
+            window.location.protocol + "//" +
+            window.location.hostname + (
+              window.location.port ?
+              ':' + window.location.port:
+              ''
+            );
+        }
+
+        return fetch(origin + window.location.pathname + 'swapi', {
```
Comment (asiandrummer) [inline test/index.html]: Noticed we're trying to use `swapi` schema and proxy the swapi parse app we're using as an example - is this intentional? Don't we want to use the mocked server schema, how we had it in the `./examples` directory?

Speaking of that, maybe a right path is to define a test schema used in `GraphiQL` testing and use here, and do the proxying you seem to be doing in the `./test/server.js` in examples. What do you think?

Comment (joelgriffith) [inline test/index.html]: Thanks for the comments! We could probably just export the schema in the `example` directory and I can just use it here. Probably the most flexible solution I can think of.

**Pull request closed without merging**